app/models/edm/rights.rb

```diff
@@ -17,12 +17,17 @@ def normalise(string)
       end
 
       def for_api_query(value)
-        registry.detect { |rights| rights.api_query == value }
+        registry.detect { |rights| value.match(rights.pattern) }
+      end
+
+      def from_api_query(value)
+        simple_value = value.to_s.tr('?*', '')
+        registry.detect { |rights| simple_value.match(rights.pattern) }
       end
     end
 
     def api_query
-      super || pattern + '*'
+      super || pattern.gsub(/\(.*\)\?|.\?/, '*') + '*'
```

Comment:
As api_query would now always be derived and never declared explicitly in rights.yml, the call to `super` can be removed.